syncd/NotificationProcessor.cpp

```diff
@@ -397,7 +397,20 @@ void NotificationProcessor::process_on_port_state_change(
          * switch vid.
          */
 
-        oper_stat->port_id = m_translator->translateRidToVid(oper_stat->port_id, SAI_NULL_OBJECT_ID);
+        SWSS_LOG_DEBUG("Port RID %s state change notification", 
+                sai_serialize_object_id(oper_stat->port_id).c_str());
+
+        if (false == m_translator->tryTranslateRidToVid(oper_stat->port_id, oper_stat->port_id))
+        {
+            oper_stat->port_id = SAI_NULL_OBJECT_ID;
+        }
+        /*
+         * port may be in process of removal. OA may recieve notification for VID either
```

Comment:
start with Capital letter, and add empty line before startin comment between 406 and 407

Reply:
Ok.